modules/core/test/v2/fixtures/coins/eos.ts

```diff
@@ -208,7 +208,7 @@ const explainTransactionInputNative = {
   headers: {
     ref_block_prefix: 686520069,
     ref_block_num: 3530,
-    expiration: '2021-10-04T16:41:53'
+    expiration: '2021-10-04T16:41:53',
```

Comment:
oh wow I like how we are modifying test. At the same time, was this file breaking test before?

Reply:
or does it mean the encoding and decoding is breaking our previous format?

Reply:
@shihao-guo no - tests were always passing here (i think most of these changes were formatting from prettier lol). for most use cases, using the standard TextDecoder is fine.

there are only problems when using it within an Electron app.

Reply:
i would recommend revert the change to the test file here if we can't explain why it needs to change

Reply:
looked it over again - all the changes are adding a trailing comma to JSON dicts.

the only intentional change is adding `unsignedTransaction`

Reply:
got it. I was referring to those commas added by prettier

Reply:
reverted commas